app/assets/stylesheets/analytics/analytics.scss

```diff
@@ -0,0 +1,72 @@
+@import url(https://fonts.googleapis.com/css?family=Nunito:300,400);
+
+$white: #ffffff;
+$accent: #30385f;
+$dark: #e3e3e3;
+$medium: #ebebeb;
+$light: #f5f5f5;
+
+.style {
+  font-family: 'Nunito', sans-serif;
+  font-weight: 200;
+}
+//*************************** CHARTS ***************************************
+
+.chart {
+  width: 100%;
+}
+
+.stacked-bar {
+  height: 767px;
+}
+
+.pie {
+  height: 367px;
+}
+
+.timeline-chart {
+  height: 288px;
+}
+
+// ************************ GRID FORMATTING *********************************
+
+.fit {
+  margin-bottom: 0;
+  padding-bottom: 0;
+  padding-top: 0;
+}
+
+.dark {
+  background-color: $dark;
+}
+
+.medium {
+  background-color: $medium;
+}
+
+.light {
+  background-color: $light;
+}
+
+// ************************ TEXT FORMATTING *********************************
+
+
+.heading {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.